### PR TITLE
Add basic financial data: stock RIC

### PR DESCRIPTION
**Pull request by thomascharbonnel**
I've added:

- [x] Basic stock data (SEHK + Nikkei random RIC generation)
- [x] USAGE.md docs if applicable
- [x] CHANGELOG.md

### CODE REVIEW

**Comment by thomascharbonnel** [comment]
So, this is very basic as a start for some stock data.
I have put 4 doctests per method (as mentioned in another PR). One issue is that the second method's (ticker(:reuters, :nikkei225)) doctests are ignored, do you have an idea why? @vbrazo

**Comment by thomascharbonnel** [comment]
Update Changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Change log itself follows [Keep a CHANGELOG](http://keepachangelog.com) format.
 - Added: Support for `Armenian Language` [[@hovikman][]]
 - Added: `Faker.Vehicle` [[@daytonn][]]
 - Added: Elixir 1.9 support [[@igas][]]
+* Added: Category `Finance` [[@thomascharbonnel][]]
```
Comment (vbrazo) [inline CHANGELOG.md]: Since you're first contributor, you must include your name in the contributors area at the end of this file.

Comment (thomascharbonnel) [reply CHANGELOG.md]: @vbrazo Thank you, updated now :)

**Comment by thomascharbonnel** [comment]
@vbrazo ?

**Pull request merged**